libsolidity/Compiler.h

```diff
@@ -41,6 +41,9 @@ class Compiler: private ASTConstVisitor
 	bytes getAssembledBytecode() { return m_context.getAssembledBytecode(m_optimize); }
 	bytes getRuntimeBytecode() { return m_runtimeContext.getAssembledBytecode(m_optimize);}
 	void streamAssembly(std::ostream& _stream) const { m_context.streamAssembly(_stream); }
+	/// Getters for compiler contexts. Only for testing purposes.
```

Comment:
Please test differently. In the end, we will need a way to get the `AssemblyItem`s together with Locations anyway. Can you return those? I.e. just as we do `getAssembledBytecode` and `getRuntimeBytecode`, can we have `getAssemblyItems` and `getRuntimeAssemblyItems`?